bin/pulsar

```diff
@@ -209,6 +217,11 @@ elif [ $COMMAND == "global-zookeeper" ]; then
     # Allow global ZK to turn into read-only mode when it cannot reach the quorum
     OPTS="${OPTS} -Dreadonlymode.enabled=true"
     exec $JAVA $OPTS -Dpulsar.log.file=$PULSAR_LOG_FILE org.apache.pulsar.zookeeper.GlobalZooKeeperStarter $PULSAR_GLOBAL_ZK_CONF $@
+elif [ $COMMAND == "configuration-store" ]; then
```

Comment:
The only real difference between "global-zookeeper" and "configuration-store" is that the log file changes. Do we care about keeping it the same? In any case, they should both call ConfigurationStoreStarter.

Reply:
Thanks. Yes, the log files are different. if user use old command, it is better to use old log file name; and use new log file name, while use new command.

Reply:
Ok. Both branches should use ConfigurationStoreStarter though. GlobalZooKeeperStarter should probably be deleted.

Reply:
Thanks, will use both ConfigurationStoreStarter here. 
GlobalZooKeeperStarter marked as Deprecated, we could remove it in later release.